File: slo.py
#!/usr/bin/env python3
class Point:
    def __init__(self, x, y):
        self.x = x
        self.y = y

total = 0
for _ in range(1000000):
    p = Point(2, 3)
    total = total + p.x + p.y

print(total)

File: slo.swift
#!/usr/bin/env swift
class Point {
  var x = 0, y = 0
  init(x: Int, y: Int) {
    self.x = x
    self.y = y
  }
}

var total = 0
for _ in 0..<1_000_000 {
  let p = Point(x: 2, y: 3)
  total = total + p.x + p.y
}

print(total)
